src/components/Match/matchPages.jsx

```diff
@@ -35,6 +36,19 @@ import Chat from './Chat';
 import { StyledFlexContainer, StyledFlexElement } from './StyledMatch';
 
 const matchPages = [Overview, {
+  name: strings.tab_drafts,
+  key: 'draft',
+  content: match => (
+    <div>
+      {match.game_mode === 2 &&
```

Comment:
what happens if the match is not game mode 2 (captains mode?)  Does it just show a blank page?

Reply:
Yea, I'm not sure what to display in that case, since atm draft_timings only returns data for all picks. I'd rather just not show the page at all (unless browsing a CM game), but wasn't sure how to do that.